### PR TITLE
fix: Remove `Real`

### PR DESCRIPTION
**Pull request by Rhuagh**
This change is [Reviewable](https://reviewable.io/reviews/rhuagh/rhusics/50)

### CODE REVIEW

**Comment by Rhuagh** [comment]
Fixes #43.

File: src/collide/narrow.rs [hunk 1]
```diff
@@ -175,11 +175,10 @@ mod tests {
     use collision::algorithm::minkowski::GJK2;
     use collision::primitive::Rectangle;
 
-    use Real;
     use collide::*;
     use collide::narrow::NarrowPhase;
 
-    fn transform(x: Real, y: Real, angle: Real) -> Decomposed<Vector2<Real>, Basis2<Real>> {
+    fn transform(x: f32, y: f32, angle: f32) -> Decomposed<Vector2<f32>, Basis2<f32>> {
```
Comment (torkleyy) [inline src/collide/narrow.rs]: Why isn't this generic?

Comment (Rhuagh) [reply src/collide/narrow.rs]: It's just for tests (it's in a tests mod). I could make it generic, but it's gonna be forced to f32 three lines of code later anyways, so I didn't see the need.

**Comment by Rhuagh** [comment]
bors r=torkleyy

**Pull request merged**